gradient/api_sdk/workspace.py

```diff
@@ -1,44 +1,14 @@
 import os
 import tempfile
 
-import progressbar
-from requests_toolbelt.multipart import encoder
-
 from gradient import cliutils, cli_constants
-from gradient.api_sdk import s3_uploader
+from gradient.api_sdk import s3_uploader, archivers
 from gradient.cli_constants import CLI_PS_CLIENT_NAME
```

Comment:
Don't import from outside of the "SDK part" - it'll make it much harder to make it a separate package in the future and may cause problems because of circular imports.